corehq/apps/reports/tasks.py

```diff
@@ -146,20 +144,6 @@ def get_domains_to_update_es_filter():
     )
 
 
-def is_app_active(app_id, domain):
-    return app_has_been_submitted_to_in_last_30_days(domain, app_id)
-
-
-@periodic_task(run_every=crontab(hour="2", minute="0", day_of_week="*"), queue='background_queue')
-def apps_update_calculated_properties():
-    query = AppES().is_build(False).values_list('_id', 'domain', scroll=True)
```

Comment:
Exciting - one step closer to removing apps from ES.  Feels like it's not worth the effort to store them there